src/Components/TableComponents/TextInputBasic.js

```diff
@@ -12,14 +12,16 @@ import {
 import { CheckCircleIcon, InProgressIcon } from '@patternfly/react-icons';
 
 export const TextInputBasic = ({
-  textValue,
+  hasWriteAccess,
+  isLoading,
+  isWriteAccessLoading,
   setTextValue,
+  textValue,
   verifiedStatus,
-  isLoading,
 }) => {
   return (
     <Flex direction={{ default: 'column' }}>
-      <TextInputGroup>
+      <TextInputGroup isDisabled={!isWriteAccessLoading && !hasWriteAccess}>
```

Comment:
This has to be moved down into the <TextInput> Component to match the mocks. Right now the box doesnt gray out. so this line would just be `<TextInputGroup>`
and beneath, under line 30 would be 
`<TextInput
          isDisabled={!isWriteAccessLoading && !hasWriteAccess}`